packages/expo-updates/android/src/main/java/expo/modules/updates/db/UpdatesDatabase.kt

```diff
@@ -49,56 +49,52 @@ abstract class UpdatesDatabase : RoomDatabase() {
       return instance!!
     }
 
-    val MIGRATION_4_5: Migration = object : Migration(4, 5) {
-      override fun migrate(database: SupportSQLiteDatabase) {
-        // https://www.sqlite.org/lang_altertable.html#otheralter
+    private fun SupportSQLiteDatabase.runInTransactionWithForeignKeysOff(block: SupportSQLiteDatabase.() -> Unit) {
+      // https://www.sqlite.org/lang_altertable.html#otheralter
+      try {
+        execSQL("PRAGMA foreign_keys=OFF")
```

Comment:
This is a code change, but shouldn't this go outside of the try-block, similar to how beginTransaction() is outside of the nested try-block whose finally-block ends the transaction?

Reply:
@jkhales had a similar comment previously: https://github.com/expo/expo/pull/15401#discussion_r761902404
@esamelson also commented: https://github.com/expo/expo/pull/15401#discussion_r763528313

Reply:
I prefer to leave it as is for now and change them in a follow-up